src/frontend/src/eth/components/fee/FeeDisplay.svelte

```diff
@@ -41,19 +46,37 @@
 
 		clearTimeout(timer);
 	});
+
+	const balance = nonNullish($feeIdStore)
+		? $balancesStore?.[$feeIdStore]?.data ?? BigNumber.from(0n)
+		: BigNumber.from(0n);
+
+	let insufficientFeeFunds = false;
+	$: insufficientFeeFunds = nonNullish(fee) && balance.lt(fee);
 </script>
 
 <label for="balance" class="font-bold px-4.5"
 	>Max fee <small>(likely in &lt; 30 seconds)</small>:</label
 >
-<div id="balance" class="font-normal px-4.5 mb-4 break-all" style="min-height: 24px">
+<div id="balance" class="font-normal px-4.5 mb-4 break-all min-h-10">
 	{#if nonNullish(fee)}
 		<div transition:fade>
 			{formatToken({
 				value: fee,
 				displayDecimals: EIGHT_DECIMALS
 			})}
 			{$feeSymbolStore ?? ''}
+			{#if insufficientFeeFunds}
+				<p transition:slide={{ duration: 250 }} class="text-cyclamen text-xs">
```

Comment:
Other errors are no shown with `text-xs`, is that a new UI requirement?

Reply:
with the normal size (the one used for validations of the inputs for example) it was too big; I showed it to @artkorotkikh-dfinity  and he said it was ok like this, but maybe he wants to revisit it?

Reply:
@AntonioVentilii-DFINITY  I don't really have a strong opinion on this – for consistency we can use the same font size as we have in other places.

Reply:
see below:

<img width="557" alt="Screenshot 2024-07-29 at 11 55 00" src="https://github.com/user-attachments/assets/b513a555-b2b7-4a23-963f-e1ad46690968">

Reply:
It's fine, thanks!

Reply:
when there is no error, it will become like this

<img width="554" alt="Screenshot 2024-07-29 at 11 57 34" src="https://github.com/user-attachments/assets/f8e87e60-a3a1-4562-a7bb-e9c35d9f84ac">

Reply:
adjusted